### PR TITLE
Simplify health checks

### PR DESCRIPTION
**Pull request by turing85**

### CODE REVIEW

File: core/runtime/src/main/java/io/quarkus/artemis/core/runtime/health/ServerLocatorHealthCheck.java [hunk 1]
```diff
@@ -15,68 +16,34 @@
...
-            namesToIgnore.addAll(buildTimeConfigs.configs().keySet());
-            Map<String, ServerLocator> locatorNamesFromArc = ArtemisUtil.extractIdentifiers(ServerLocator.class, namesToIgnore);
-            for (var entry : locatorNamesFromArc.entrySet()) {
-                ServerLocator locator = entry.getValue();
-                if (locator != null) {
-                    serverLocators.put(entry.getKey(), locator);
-                }
-            }
-        }
+            @SuppressWarnings("CdiInjectionPointsInspection") ArtemisHealthSupport support,
```
Comment (gastaldi) [inline core/runtime/src/main/java/io/quarkus/artemis/core/runtime/health/ServerLocatorHealthCheck.java:32]: Why `@SuppressWarnings` is needed here? Can't we add `@Inject` in the construtor to make the compiler happy?

**Review by gastaldi — APPROVED** [review]
Nice

File: core/runtime/src/main/java/io/quarkus/artemis/core/runtime/health/ServerLocatorHealthCheck.java (hunk 1 above)
Comment (turing85) [reply core/runtime/src/main/java/io/quarkus/artemis/core/runtime/health/ServerLocatorHealthCheck.java:32]: Nope. This is more of an IntelliJ-thing. For whatever reason, IntelliJ is not able to see the bean. I normally suppress those warnings when I know that the beans are present.

**Comment by turing85** [comment]
@zhfeng feel free to merge this onto you branch when you are satisfied with the changes. This should also trigger a re-run of you branch's pipeline (which failed for some technical reason).

**Review by zhfeng — APPROVED** [review]

**Pull request merged**